screens/EventCreation/EventCreationScreen.tsx

```diff
@@ -126,16 +138,28 @@ const EventCreationScreen = ({ navigation }: EventCreationScreenProps) => {
             onChangeText={setTitle}
           />
           <Text style={[styles.tagsTitle, globalStyles.text]}>
-            Choose up to three tags
+            Choose up to three interests
           </Text>
 
-          <Pressable onPress={() => alert("Coming in the next sprint")}>
+          <Pressable
+            onPress={() => {
+              navigation.navigate("Interests")
+              setSelectedInterests([])
+            }}
+          >
             <View style={styles.tags}>
-              {[...Array(3)].map((_, index) => (
-                <View style={styles.addTag} key={index}>
+              <View style={styles.addTag}>
```

Comment:
As can be seen in the demo, the icon is not centered inside the button. `addTag` should contain `alignItems: "center"` and `justifyContent: "center"` together with some `paddingVertical` and `paddingHorizontal`